packages/core/contracts/optimistic-asserter/implementation/escalation-manager/FullPolicyEscalationManager.sol

```diff
@@ -0,0 +1,206 @@
+ * @dev If nothing is configured using the setters and configureEscalationManager method upon deployment, the
+ * FullPolicyEscalationManager will return a default policy with all values set to false.
+ */
+contract FullPolicyEscalationManager is BaseEscalationManager, Ownable {
+    struct ArbitrationResolution {
+        bool valueSet; // True if the resolution has been set.
+        bool resolution; // True or false depending on the resolution.
+    }
+
+    event EscalationManagerConfigured(
+        bool blockAssertion,
+        bool arbitrateViaEscalationManager,
+        bool discardOracle,
+        AssertionBlockMode assertionBlockMode
+    );
+
+    event ArbitrationResolutionSet(bytes32 indexed identifier, uint256 time, bytes ancillaryData, bool resolution);
+
+    event DisputeCallerWhitelistSet(address disputeCaller, bool whitelisted);
+
+    event AssertingCallerWhitelistSet(address assertingCaller, bool whitelisted);
+
+    event AsserterWhitelistSet(address asserter, bool whitelisted);
+
+    AssertionBlockMode public assertionBlockMode; // The mode for blocking assertions.
+
+    bool public arbitrateViaEscalationManager; // True if we should arbitrate via the escalation manager instead of the DVM.
+
+    bool public discardOracle; // True if we should ignore the resolution of a potential dispute arbitrated by the Oracle (DVM or EM).
+
+    bool public validateDisputers; // True if we should restrict who can file disputes via a whitelistedDisputeCallers list.
+
+    mapping(bytes32 => ArbitrationResolution) public arbitrationResolutions; // Arbitration resolutions for a given identifier, time, and ancillary data.
+
+    mapping(address => bool) public whitelistedDisputeCallers; // Whitelisted disputer that can file disputes.
+
+    mapping(address => bool) public whitelistedAssertingCallers; // Whitelisted assertingCallers that can assert prices.
+
+    mapping(address => bool) public whitelistedAsserters; // Whitelisted asserters that can assert prices.
+
+    /**
+     * @notice Gets the price for identifier and time if it has already been requested and resolved.
+     * @dev If the price is not available, the method reverts.
+     * @param identifier uniquely identifies the price requested.
+     * @param time unix timestamp of the price request.
+     * @param ancillaryData arbitrary data appended to a price request to give the voters more info from the caller.
+     * @return int256 representing the resolved price for the given identifier and timestamp.
+     * @dev This function replicates the interface of the corresponding DVM function to allow the user to use his own
+     * dispute arbitration system when arbitrating via the escalation manager in a DVM-compatible manner. Refer to the
+     * UMA Voting and VotingV2 contracts for further details.
+     */
+    function getPrice(
+        bytes32 identifier,
+        uint256 time,
```

Comment:
Nit: can this function go first?

Reply:
This is the first function defined in the file. Not sure if I understand

Reply:
Sorry. Did the review on my phone. I mean put the get assertion policy function first. Put the most important function at the top :)

Reply:
Oh, ok, makes sense. Fixed!